projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/collection/write-collection-field.component.ts

```diff
@@ -332,6 +335,29 @@ export class WriteCollectionFieldComponent extends AbstractFieldWriteComponent i
     return !!id && !this.getCollectionPermission(this.caseField, 'allowDelete');
   }
 
+  public checkComplexFieldReadOnly(): boolean {
+    return this.checkComplexFieldsReadOnly(this.caseField);
+  }
+
+  checkFieldType(caseField: CaseField): CaseField[] {
```

Comment:
@Josh-HMCTS can you specify access modifier

Reply:
@RiteshHMCTS I have added access modifiers to both of the review functions